web/src/topic_list.ts

```diff
@@ -82,6 +83,7 @@ export function zoom_out(): void {
     const parent_widget = widget.get_parent();
 
     rebuild_left_sidebar(parent_widget, stream_id);
+    topic_filter_pill_widget?.clear(true);
```

Comment:
`topic_filter_pill_widget?.clear` should be called before we rebuild left sidebar.